web/src/resolvers/edit-project.resolver.ts

```diff
@@ -10,9 +10,11 @@ import { ProjectService } from '../services/project.service';
 })
 export class EditProjectResolver implements Resolve<boolean> {
 
-    constructor(private authService: AuthenticationService,
-                private projectService: ProjectService,
-                private router: Router) {
+    constructor(
+        private authService: AuthenticationService,
+        private projectService: ProjectService,
+        private router: Router
+        ) {
```

Comment:
I think this is more readable. Does the IDE not do this automatically?

```typescript
constructor(
   private authService: AuthenticationService,
   private projectService: ProjectService,
   private router: Router
) { }
```

Reply:
Yes Id does it automatically , I have updated it initially. Will take Id formatting going forward

Reply:
👍